### DISCUSSION TITLE
Applications

### DISCUSSION THREAD

**Discussion by RainerHeintzmann** [JuliaMath/NFFT.jl#88] [Show and tell]
Maybe this is a good place to collect applications based on `NFFT`?

Wrt. the earlier issue/discussion on resampling based on NFFTs, you can have a look at the branch:

https://github.com/bionanoimaging/FourierTools.jl/tree/resample_var

,which contains the functions `nfft_nd()` etc. and also the application `rescale_nfft()` which allows to **resample (regrid) data** from (almost) arbitrary grids or onto (almost) arbitrary grids as long as the topology is representable by arrays.
It may soon be part of the distributed version of the FourierTools package.

Here is an example of appying a spiral-shaped coordinate transform to a test image:
![image](https://user-images.githubusercontent.com/7559915/153833746-27801532-6057-49de-a5fd-b88bb8b8bd4a.png)

**Comment by tknopp** [comment]
I would love to have some application section (e.g. within the documentation?) where we collect applications of the NFFT. I have to admit that I don't have the complete picture and in turn would learn quite a bit from that page. I know the NFFT from these domains:
* Magnetic resonance imaging with non-equidistant sampling trajectories
* Exploiting the Fourier Slice Theorem (-> Computed Tomography)
* Optical coherence tomography reconstruction (prisms spread the light non-equidistantly)
